discovery/ionos/server_test.go

```diff
@@ -65,12 +65,11 @@ func TestIONOSServerRefresh(t *testing.T) {
 			"__meta_ionos_server_id":                "b501942c-4e08-43e6-8ec1-00e59c64e0e4",
 			"__meta_ionos_server_ip":                ",85.215.233.222,",
 			"__meta_ionos_server_lan_id":            ",1,",
+			"__meta_ionos_server_lan_ip_1":          ",85.215.233.222,",
```

Comment:
note for this file: `io/ioutil` is in the blacklist: Use corresponding 'os' or 'io' functions instead. (depguard)

Reply:
I've replaced the package.